packages/core/src/components/ui/Checkbox/Checkbox.js

```diff
@@ -23,6 +23,7 @@ const Checkbox = forwardRef(function Checkbox(props, ref) {
     error,
     small,
     helpMessage,
+    helpMessageClassname,
```

Comment:
please rename to `helpMessageClassName` to match with naming convention used in project  `headerClassName/cellClassName`